### PR TITLE
License added

### LINKED ISSUE #58: chore: add licence to the repository
What is the license for this repository, please?

Thank you.

### PR DESCRIPTION
**Pull request by MukulKolpe**
Fixes #58

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 Deno Land
```
Comment (satyarohith) [inline LICENSE:3]:
```suggestion
Copyright (c) 2018-2021 the Deno authors
```

**Comment by satyarohith** [comment]
@MukulKolpe, you need to sign the CLA before we can merge the PR. See https://github.com/denoland/manual/pull/59#issuecomment-903679678

**Comment by MukulKolpe** [comment]
@satyarohith, done👍

**Review by satyarohith — APPROVED** [review]
LGTM. Thanks

**Pull request merged**